web-ui/eslint.config.js

```diff
@@ -31,6 +41,13 @@ export default [
       'react-hooks/exhaustive-deps': 'warn',
       ...vitestPlugin.configs.recommended.rules,
       'vitest/expect-expect': 'off',
+      '@typescript-eslint/no-unused-vars': 'off',
+      '@typescript-eslint/ban-ts-comment': 'off'
     }
+  },
+  {
+    // disable type-aware linting on JS and JSX files
```

Comment:
Why do we want to do this?

Reply:
That was protoype code. It's been removed.